libs/vm_modules/tests/unit/math.cpp

```diff
@@ -32,15 +39,148 @@ class MathTests : public ::testing::Test
 TEST_F(MathTests, abs_test)
 {
   static char const *TEXT = R"(
-    function main()
-      print(abs(-1));
+    function main() : Int32
```

Comment:
Throughout this file you seem to execute scripts more than once: there are assertions of the form `ASSERT_TRUE(toolkit.Run());` followed by `ASSERT_TRUE(toolkit.Run(&res));`. The former are redundant.